### DISCUSSION TITLE
Be the first to test the new Collabora Online

### DISCUSSION THREAD

**Discussion by pedropintosilva** [CollaboraOnline/online#3461] [Announcements]
📢 tl;dr: Join us and smoke test what it will be the upcoming new stable version of Collabora Online

:mega:  :calendar: Tomorrow **Oct 26, 12:00 CEST** Collaborative Testing session

![smoke-test-banner](https://user-images.githubusercontent.com/65948705/138545341-c5dbb6d1-b3a2-4126-ae9d-48bc4990ffe9.jpg)

## 👋 We need you!
  Become part of the next version's improvements by smoke testing the current alpha
* :package: What: Alpha 1 (that was already released AKA 1st alpha of 21.11 version)
* :calendar: When: Still to be decided
* :computer: How:
  * Using a jitsi room: https://meet.jit.si/COOLSmokeTest
    * With a short introduction and explanation
    * Everyone will work collaboratively around the same spreadsheet filling the blanks according to their OS, web browser, etc
  * OR by doing it alone
    * By filling up the bits you want in [this spreadsheet](https://staging.eu.collaboraonline.com/nextcloud/index.php/s/B7W9LLFcWZQn64G)
    * And by [reporting any bugs](https://github.com/CollaboraOnline/online/issues/new?assignees=&labels=smoke+bug%2C+unconfirmed&template=bug_report_smoke-test.md&title=) 
* Goal: You will have fun; Help improve Collabora Online and get credited in the process 💪
* Report your bugs using [this bug type](https://github.com/CollaboraOnline/online/issues/new?assignees=&labels=smoke+bug%2C+unconfirmed&template=bug_report_smoke-test.md&title=)

:heavy_heart_exclamation:  **Interested? Request access:**
* Send email to alpha-1@collaboraoffice.com if you wish to have a testing account 
  * An NC account with Collabora Online will be created for you ;) at https://staging.eu.collaboraonline.com/nextcloud
* Don't want to create an account?
  * Informal Smoke test spreadsheet to help you in your tests: https://staging.eu.collaboraonline.com/nextcloud/index.php/s/B7W9LLFcWZQn64G
    * Demo documents that you can use: https://staging.eu.collaboraonline.com/nextcloud/index.php/s/nM8ramjZdk3SQKA
  * If you have your own Nextcloud or ownCloud you can actually just use https://staging.eu.collaboraonline.com in the settings > under admin > Collabora Online Development Edition this why you can also test other things

For any question just ask at:
* https://meet.jit.si/COOLSmokeTest (I will try to be almost all the time there during the day)
* or alpha-1@collaboraoffice.com
* or https://collaboraonline.github.io/post/communicate/#get-in-touch

**Comment by dnegreira** [comment]
I am interested in testing - but not on creating an account, if you can send me the links I would prefer it that way.

**Reply by pedropintosilva** [reply]
Cool! : ) if you have your own Nextcloud or ownCloud you can actually just use `https://staging.eu.collaboraonline.com` in the settings > under admin > Collabora Online Development Edition

**Reply by pedropintosilva** [reply]
@dnegreira I have updated the description

**Reply by dnegreira** [reply]
Neat, thanks!
I will try the Nextcloud way, and if that fails (probably due to my setup) I will use the links.

**Reply by pedropintosilva** [reply]
@dnegreira hm you shouldn't  encounter any hiccup:
![image](https://user-images.githubusercontent.com/65948705/138674967-dae886ad-75cf-4d02-b468-20344734e1f6.png)

**Reply by pedropintosilva** [reply]
1. Avatar > + apps
2. Search for Collabora Online (the connector , see the description)
3. Install
4. Go to settings
5. Click in the app on the left side
6. Fill in the `https://staging.eu.collaboraonline.com/`

**Reply by dnegreira** [reply]
That works like a charm! I am able to use the staging server from my nextcloud instance.
How do I now access the smoke test spreadsheet from my nextcloud instance?

**Reply by pedropintosilva** [reply]
@dnegreira hm I can only think of two ways:
* If your Nextcloud has the federative option, you could send me the id and I would invite you to the folder
* Or just simply download the spreadsheet and the demo docs folder and upload to your NC : p

Ah or editing the spreadsheet using the public link and testing with your own documents in your NC

**Reply by dnegreira** [reply]
I think the option of editing the public link spreadsheet + testing with my own documents in my NC would be the best option.
Is this something you are looking to test as well?

**Comment by pedropintosilva** [comment]
:mega:  :calendar: Tomorrow **Oct 26, 12:00 CEST** Collaborative Testing session

**Comment by pedropintosilva** [comment]
in 4 min : https://meet.jit.si/COOLSmokeTest

**Reply by PeterHagen** [reply]
Sorry, I mist it...

**Reply by pedropintosilva** [reply]
@PeterHagen no problem, feel free to test on your own if you so which and create [bug report](https://github.com/CollaboraOnline/online/issues/new?assignees=&labels=smoke+bug%2C+unconfirmed&template=bug_report_smoke-test.md&title=) : ) or feel free to ping me anytime in the chats or via email

**Reply by pedropintosilva** [reply]
@PeterHagen : Let me know when you are available and we can do a collaborative testing session ; )
